direct-messages/README.md

```diff
@@ -97,6 +140,110 @@ var content = {
 
 ---
 
+## 2. Mapping `pobox_id` to `recp_key`
+
+The purpose of a P.O. Box is to be able to send a direct message which can be read only by holders of
+the P.O. Box secret key.
+While this allows messaging with (potentially) a group of feeds, it is **not a "Private Group"**:
+- it allows outisiders to message _in_ to a some grouping of feeds _(private groups do no allow this)_
+- this is built with asymetric keys _(private groups use symmetric keys)_
+
+The description below also seems _like_ DMs with a `feed_id`, but:
+- it allows for messages to a P.O. Box that the sender can not later decrypt _(`feed_id` DMs do not support this)_
+
+The following design is made so that the `recps` field is the same no matter who is sending the message.
+
+There are two cases:
+    - **2a. Outside messaging a P.O. Box**
+    - **2b. P.O. Box member replies**
+
+### 2a. Outsider messaging a P.O. Box
+
+```js
+const request = {
+  type: 'post',
+  text: 'hello, my name is Cherese, can i join your group please?'
+  recps: [
+    '$D6YvQIFYo3lQhmNbEjoQc7sLiPcDJdzQqF/ye+QM09g=.pobox-1', // pobox_id
```

Comment:
TODO: decide the format of this, add to TFK spec/ BFE thing

- happy enough with the suffix
- what is the **prefix**
    - @keks pointed out that `@` is currently references a feedId (which is defines a signed log) but also awkwardly currently also defines a DM location (because we derive Diffie-Hellman keys from our signing keys)... so perhaps it would be nice to have a new sigil for encrypting-to.
    - maybe we stick with @ being about a UX thing which you really want to be talking to a person...., arg
    
    
Need to mention that there's a `pobox_sk` and `pobox_pk` and this ID is `$<pobox_pk.toBase64>.pobox-1`